Source/Charts/Charts/BarLineChartViewBase.swift

```diff
@@ -927,22 +927,17 @@ open class BarLineChartViewBase: ChartViewBase, BarLineScatterCandleBubbleChartD
 
             var foundScrollView = scrollView as? NSUIScrollView
             
-            if foundScrollView !== nil && !foundScrollView!.nsuiIsScrollEnabled
+            if !(foundScrollView?.nsuiIsScrollEnabled ?? true)
             {
                 foundScrollView = nil
             }
             
             var scrollViewPanGestureRecognizer: NSUIGestureRecognizer!
             
-            if foundScrollView !== nil
```

Comment:
replace the loop with
```
        let scrollViewPanGestureRecognizer = foundScrollView?.nsuiGestureRecognizers?.first {
            $0 is NSUIPanGestureRecognizer
        }

```

Reply:
I'm not confident in using `.first` here to achieve the same effect

Reply:
Why not? The code you've replaced the for loop with is not the same. In the for loop, we escape after we find the first `NSUIPanGestureRecognizer`, in your code it will continue to loop through and find more.

Reply:
Rather than leave it the old way (using our own implementation of `first`) why not simply use the built-in method and replace 5 lines with 2.

Reply:
done